operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md

```diff
@@ -0,0 +1,185 @@
+- [Example session template input file](#example-session-template-input-file)
+- [Creating a BOS session using the new template](#creating-a-bos-session-using-the-new-template)
+
+The Scalable Boot Projection Service (SBPS) is an optional provider for the `rootfs` on compute nodes.
+
+## Boot set `rootfs_provider` parameter
+
+The following value needs to be set in the boot set of the session template in order to make SBPS the `rootfs` provider:
+
+- `"rootfs_provider":` Set to `"sbps"`
+
+## Boot set `rootfs_provider_passthrough` parameters
+
+For SBPS, the `rootfs_provider_passthrough` boot set parameter is customized according to the following format:
+
+```text
+rootfs_provider_passthrough=<transport>:<schema version>:<IQN Domain>:<DNS SRV record reference>:<client discovery timeout in seconds>
+```
+
+The following values need to be set in the boot set of the session template in order to make SBPS the `rootfs` provider:
+
+- `"rootfs_provider":` Set to `"sbps"`
+- To use the Node Management Network (NMN) for content projection,
+  - `"rootfs_provider_passthrough"`: Set to `"sbps:v1:iqn.2023-06.csm.iscsi:_sbps-nmn._tcp.odin.hpc.amslabs.hpecorp.net:300"`
+- To use the High Speed Network (HSN) for content projection,
+  - `"rootfs_provide_passthrough"`: Set to `"sbps:v1:iqn.2023-06.csm.iscsi:_sbps-hsn._tcp.odin.hpc.amslabs.hpecorp.net:300"`
+
+Note that `iqn.2023-06.csm.iscsi` is the IQN domain.
+
+The variables used in this parameter represent the following:
+
+### `<transport>`
+
+File system network transport. For example, `sbps` or `dvs`.
+
+Set to `sbps` to use SBPS. Can be left as an empty string to use the default value `dvs`.
+
+### `<schema version>`
+
+The version of the SBPS schema. `v1` is the only supported version.
+
+### `<IQN Domain>`
+
+The domain name portion of the iSCSI Qualified Name (IQN).
+
+For example, an IQN might look like this:
+
+```text
+iqn.2023-06.csm.iscsi:ncn-w002
+```
+
+In this example:
+
+- `iqn.2023-06.csm.iscsi` is the IQN domain.
+- `ncn-w002` is a unique identifier for the storage device.
+
+The IQN Domain helps in ensuring unique identification of iSCSI targets within a given namespace, allowing for proper routing and management of storage resources over the network
+
+For SBPS, only the domain portion of the IQN needs to be supplied, not the entire IQN.
+
+### `<DNS Service (SRV) record reference>`
+
+A DNS SRV (Service) record is a type of Domain Name System (DNS) resource record used to specify information about services offered by a particular domain.
+
+Here's how a DNS SRV record is structured:
+
+- Service: Specifies the symbolic name of the service.
+- Protocol: Specifies the transport protocol of the service, such as TCP or UDP.
+- Name: The domain name for which this record is valid.
+- Other elements were omitted for clarity and brevity.
+
+For example, DNS SRV record might look like this:
+
+```text
+_sbps-hsn._tcp.odin.hpc.amslabs.hpecorp.net
```

Comment:
Nit, use generic system site domains, don't use internal HPE DNS spaces.

Reply:
@jeremy-duckworth , can you give me an example of a generic system site domain, please?

Reply:
Looking quickly through documentation, it appears the examples are not consistent. Here is perhaps one of the better, long standing but still not ideal, examples: 

https://github.com/Cray-HPE/docs-csm/blob/b29a340f0808f3dfd0d85e846d37ae0949977429/operations/network/dns/PowerDNS_migration.md#examples